Objective-J/Runtime.js

```diff
@@ -1020,11 +1020,26 @@ GLOBAL(method_getName) = function(/*Method*/ aMethod)
     return aMethod.method_name;
 }
 
+// FIXME: This function is deprecated and should be removed in a future release
+// Please use 'method_copyReturnType' or 'method_copyArgumentType'
 GLOBAL(method_getTypes) = function(/*Method*/ aMethod)
```

Comment:
We have jus added this method, do we need to set it as deprecated ? I'm pretty sure that not a lot of people are using it !